src/Symfony/Component/Validator/Constraints/PasswordStrength.php

```diff
@@ -26,42 +24,27 @@
 #[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
 final class PasswordStrength extends Constraint
 {
+    public const STRENGTH_VERY_WEAK = 0;
+    public const STRENGTH_WEAK = 1;
+    public const STRENGTH_REASONABLE = 2;
+    public const STRENGTH_STRONG = 3;
+    public const STRENGTH_VERY_STRONG = 4;
+
     public const PASSWORD_STRENGTH_ERROR = '4234df00-45dd-49a4-b303-a75dbf8b10d8';
-    public const RESTRICTED_USER_INPUT_ERROR = 'd187ff45-bf23-4331-aa87-c24a36e9b400';
 
     protected const ERROR_NAMES = [
         self::PASSWORD_STRENGTH_ERROR => 'PASSWORD_STRENGTH_ERROR',
-        self::RESTRICTED_USER_INPUT_ERROR => 'RESTRICTED_USER_INPUT_ERROR',
     ];
 
     public string $lowStrengthMessage = 'The password strength is too low. Please use a stronger password.';
```

Comment:
As we have a single error message for this constraint now, I suggest renaming it to `message`